OregonFoodBanks/oregon_food_banks_scraper.py

```diff
@@ -68,7 +68,7 @@ def grab_data(self):
     check_collection="services",
     dump_collection="tmpOregonFoodBanks",
     dupe_collection="tmpOregonFoodBanksFoundDuplicates",
-    data_source_collection_name="oregon_food_banks",
+    data_source_collection_name="OregonFoodBanks",
```

Comment:
I prefer `oregon_food_banks` more over `OregonFoodBanks`, Being consistent with existing values in data-sources collection in mongodb.